routes/selection.js

```diff
@@ -72,6 +72,60 @@ router.get('/:geotype/:geoid', async (req, res) => {
   }
 });
 
+router.post('/:geotype', async (req, res) => {
+  const { app } = req;
+  let { geotype } = req.params;
+  let { geoid } = req.body;
+
+  geoid = deserializeGeoid(res, geotype, geoid);
+
+  if (geotype === 'selection') {
+    try {
+      const selection =  await app.db.query('SELECT * FROM selection WHERE hash = ${geoid}', { geoid });
+
+      if (selection && selection.length > 0) {
```

Comment:
Some geewiz syntax, if you're feeling froggy

```suggestion
      if (selection?.length > 0) {
```

Reply:
this actually crashes the app
```
/Users/horatiorosa/LocalDesktop/dcp-repos/labs-factfinder-api/routes/selection.js:86
      if (selection?.length > 0) {
                    ^

SyntaxError: Unexpected token .
```

Reply:
Yikes, nvm. It's not available until [Node.js 14](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility)